ProteoformSuiteGUI/ProteoformSweet.cs

```diff
@@ -127,16 +133,15 @@ private void loadRunToolStripMenuItem_Click(object sender, EventArgs e)
                 }
 
                 // For connectivity testing --
-                //string working_directory;
```

Comment:
We don't really need this save-results section in the load and run anymore, since you've introduced a better method to do so!